java-vertexai/google-cloud-vertexai/src/main/java/com/google/cloud/vertexai/v1/EndpointServiceClient.java

```diff
@@ -0,0 +1,2178 @@
+ *
+ * <p>Many parameters require resource names to be formatted in a particular way. To assist with
+ * these names, this class includes a format method for each type of name, and additionally a parse
+ * method to extract the individual identifiers contained within names that are returned.
+ *
+ * <p>This class can be customized by passing in a custom instance of EndpointServiceSettings to
+ * create(). For example:
+ *
+ * <p>To customize credentials:
+ *
+ * <pre>{@code
+ * // This snippet has been automatically generated and should be regarded as a code template only.
+ * // It will require modifications to work:
+ * // - It may require correct/in-range values for request initialization.
+ * // - It may require specifying regional endpoints when creating the service client as shown in
+ * // https://cloud.google.com/java/docs/setup#configure_endpoints_for_the_client_library
+ * EndpointServiceSettings endpointServiceSettings =
+ *     EndpointServiceSettings.newBuilder()
+ *         .setCredentialsProvider(FixedCredentialsProvider.create(myCredentials))
+ *         .build();
+ * EndpointServiceClient endpointServiceClient =
+ *     EndpointServiceClient.create(endpointServiceSettings);
+ * }</pre>
+ *
+ * <p>To customize the endpoint:
+ *
+ * <pre>{@code
+ * // This snippet has been automatically generated and should be regarded as a code template only.
+ * // It will require modifications to work:
+ * // - It may require correct/in-range values for request initialization.
+ * // - It may require specifying regional endpoints when creating the service client as shown in
+ * // https://cloud.google.com/java/docs/setup#configure_endpoints_for_the_client_library
+ * EndpointServiceSettings endpointServiceSettings =
+ *     EndpointServiceSettings.newBuilder().setEndpoint(myEndpoint).build();
+ * EndpointServiceClient endpointServiceClient =
+ *     EndpointServiceClient.create(endpointServiceSettings);
+ * }</pre>
+ *
+ * <p>To use REST (HTTP1.1/JSON) transport (instead of gRPC) for sending and receiving requests over
+ * the wire:
+ *
+ * <pre>{@code
+ * // This snippet has been automatically generated and should be regarded as a code template only.
+ * // It will require modifications to work:
+ * // - It may require correct/in-range values for request initialization.
+ * // - It may require specifying regional endpoints when creating the service client as shown in
+ * // https://cloud.google.com/java/docs/setup#configure_endpoints_for_the_client_library
+ * EndpointServiceSettings endpointServiceSettings =
+ *     EndpointServiceSettings.newHttpJsonBuilder().build();
+ * EndpointServiceClient endpointServiceClient =
+ *     EndpointServiceClient.create(endpointServiceSettings);
+ * }</pre>
+ *
+ * <p>Please refer to the GitHub repository's samples for more quickstart code snippets.
+ */
+@Generated("by gapic-generator-java")
+public class EndpointServiceClient implements BackgroundResource {
```

Comment:
Is `EndpointServiceClient` needed? I thought that only `PredictionServiceClient` was being added.